Add installation configure of LH GuaranteedInstanceManagerCPU (#549)
harvester/docs: versioned_docs/version-v1.2/install/harvester-configuration.md

        guaranteedReplicaManagerCPU: 6
```

### `install.harvester.longhorn.default_settings.guaranteedInstanceManagerCPU`

_Available as of v1.2.2_

#### Definition

> Sets the default percentage of the total allocatable CPU on each node will be reserved for each Longhorn instance manager Pod.

Reviewer: ```suggestion
Percentage of the total allocatable CPU on each node to be reserved for each Longhorn Instance Manager pod.
```

Default: 12

Supported values: 0-12. All other values are considered 12.

> This integer value indicates what percentage of the total allocatable CPU on each node will be reserved for each instance manager Pod.
> 
> In edge scenarios where users may deploy single-node Harvester clusters, they can set this parameter to a value smaller than 12. In most scenarios, it is recommended to keep the default value for system high availability.
> 
> Before setting the value, please refer to [longhorn-guaranteed-instance-manager-cpu](https://longhorn.io/docs/1.5.5/references/settings/#guaranteed-instance-manager-cpu) for more details.

Reviewer: ```suggestion
This integer value indicates the percentage of the total allocatable CPU on each node to be reserved for each Longhorn Instance Manager pod.

Using the default value is recommended for high system availability. When deploying single-node Harvester clusters, you can specify a value less than 12.

For more information about how to set the correct value, see [Guaranteed Instance Manager CPU](https://longhorn.io/docs/1.6.0/references/settings/#guaranteed-instance-manager-cpu) in the Longhorn documentation.
```

Author: Others are updated, but the longhorn link keeps v1.5.5 as in Harvester v1.2.2, the used LH version is v1.5.5.
